README.md

```diff
@@ -96,7 +96,7 @@ more info at [Documentation](https://baianat.github.io/hooper/)
 |`wheelControl`   |true |toggle mouse wheel sliding.|
 |`keysControl`    |true |toggle keyboard control.|
 |`shortDrag`      |true |enable any move to commit a slide.|
-|`autoPlay`       |false|enable auto sliding to carousel.|
+|`autoPlay`       |false|enable auto sliding to carousel. This could be changed dynamically. If set to false at startup you have to run restartTimer() after updating for the change to take effect.|
```

Comment:
Could we call "restartTimer" internally instead? A simple watcher would do I guess?

Reply:
Sure, will use the watch.